### ISSUE TITLE
Can you  removed VAE  for Load video ？

### ISSUE THREAD

**Issue by cardenluo** [Kosinkadink/ComfyUI-VideoHelperSuite#296]
![image](https://github.com/user-attachments/assets/6dfaa02f-0a89-4f79-bdf0-f6d5d585742e)
1）“AnythingEverywhere” cannot be used. The loading of VAE has become an interference. 
       Blocking the interface with “Reroute” causes the connection line to be disconnected again.
2）The use of this VAE has not been seen in use by anyone.Can it be removed? It seems to have little use value.

**Comment by AustinMroz** [comment]
If there is a bug with cg-use-everywhere, I would appreciate a report of it. I had thought that I had ironed out all those issues and I'm unable to find anything off in my testing. If that is not the case, I would absolutely consider fixing it a high priority.
![image](https://github.com/user-attachments/assets/cfa38681-63a9-44c4-8814-e30e8156199d)


While I believe you have the best interests of the project in mind when you make the suggestion to remove the functionality,
- It is disingenuous to claim that not a single person is using it and I know multiple people who do.
- The functionality is implemented because it significantly reduces memory allocation. If this functionality were removed, there are workflows which require it that become completely impossible to execute.
- Even if the functionality was purely a quality of life feature, removing it would break existing workflows which is a matter of absolute last resort.
